_tools/import_docs.sh

```diff
@@ -20,19 +21,20 @@ git rm -rf ${OUTDIR}/docs/ ${OUTDIR}/examples/ > /dev/null
 rm -rf ${OUTDIR}/docs/ ${OUTDIR}/examples/
 
 echo adding new
-mv $TMPDIR/docs ${OUTDIR}
-mv $TMPDIR/examples ${OUTDIR}
+mv $tmpdir/docs ${OUTDIR}
+mv $tmpdir/examples ${OUTDIR}
 git add ${OUTDIR}/docs/ ${OUTDIR}/examples/ > /dev/null
-rmdir $TMPDIR
+rmdir $tmpdir
 
 echo stripping
 for dir in docs examples; do
     find ${OUTDIR}/${dir} -type f -name \*.md | while read X; do
-        sed -i \
+        sed -i .import_docs_tmp \
```

Comment:
Ahh, in versionize we actually call `gsed` on macOS - is that a better fix?  The sed logic here and there really should be ~identical.